hal/src/electron/cellular_hal.cpp

```diff
@@ -7,6 +8,36 @@
 
 static CellularCredentials cellularCredentials;
 
+static HAL_NET_Callbacks netCallbacks = { 0 };
+
```

Comment:
hm, does that mean that it's not necessary to add our typical `void* reserved` to newly added dynalib functions? That trick when a regular function is called with more arguments than it expects isn't formally allowed in C as far as I know, though it seems to be supported by all major compilers.